app/controllers/CheckYourAnswersController.scala

```diff
@@ -16,73 +16,144 @@
 
 package controllers
 
+import audit.AuditService
 import com.google.inject.Inject
 import config.FrontendAppConfig
+import connectors.DataCacheConnector
 import controllers.actions.{DataRequiredAction, DataRetrievalAction, IdentifierAction}
+import mapper.CaseRequestMapper
+import models._
+import models.requests.OptionalDataRequest
+import models.WhichBestDescribesYou.theUserIsAnAgent
+import navigation.Navigator
+import pages.{CheckYourAnswersPage, ConfirmationPage, SupportingMaterialFileListPage, UploadWrittenAuthorisationPage}
 import play.api.i18n.{I18nSupport, Lang}
-import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
-import service.CountriesService
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import service.{CasesService, CountriesService, FileService}
+import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import utils.CheckYourAnswersHelper
 import viewmodels.AnswerSection
 import views.html.check_your_answers
 
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Please can we get rid of this use of the global ExecutionContext and add an `implicit ec` to the controller instead? You can see examples in the other controllers